src/NUnitEngine/nunit-agent/Program.cs

```diff
@@ -153,16 +153,6 @@ public static int Main(string[] args)
                 log.Error("Exception in RemoteTestAgent", ex);
             }
 
-            try
-            {
-                // Unregister the channel
-                testAgencyServer.Stop();
```

Comment:
This change means the channel is no longer unregistered, as far as I can see? Should we be calling `ServerUtilities.SafeReleaseChannel()` at some point?